.changeset/tender-bottles-kneel.md

```diff
@@ -0,0 +1,13 @@
+---
+'fingerprint-pro-server-api-dotnet-sdk': patch
+---
+
+- Remove all `example` YAML attributes from all the properties due to [deprecation](https://spec.openapis.org/oas/v3.1.0#fixed-fields-19).
```

Comment:
We don't need only schema-related updates

Reply:
Fixed